schemas/org/openstax/ec/started_session/v1/avro_builder.rb

```diff
@@ -0,0 +1,14 @@
+namespace 'org.openstax.ec'
+
+import 'types/uuid'
+import 'types/date_time'
+
+record :started_session_v1 do
+  required :user_uuid, :uuid
+  required :ip_address, :string
+  required :referrer, :string
+  required :user_agent, :string
+  required :session_uuid, :uuid
+  required :session_order, :int
```

Comment:
This should be `0` always, because this is by definition the first session event.  So take this out.